cartography/intel/aws/kms.py

```diff
@@ -0,0 +1,331 @@
+import json
+import logging
+
+from botocore.exceptions import ClientError
+from policyuniverse.policy import Policy
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_kms_key_list(boto3_session, region):
```

Comment:
Can you add the `@aws_handle_regions` decorator, please?
(also below to all the functions that fetch data)

Reply:
@marco-lancini added `@aws_handle_regions` decorator in latest commit